stdlib/number.md

```diff
@@ -13,6 +13,47 @@ No other changes yet.
 from "number" include Number
 ```
 
+## Types
+
+Type declarations included in the Number module.
+
+### Number.**ParseIntError**
+
+<details disabled>
+<summary tabindex="-1">Added in <code>next</code></summary>
+No other changes yet.
+</details>
+
+```grain
+ ParseIntError = Atoi.ParseIntError {
```

Comment:
I think we might want to output something in oprint for a type manifest as this looks a little ugly.